zad5/config/routes.rb

```diff
@@ -1,10 +1,17 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+
+  get 'artists/index'
```

Comment:
ta linijka zbędna ponieważ `resources`, który jest w linii 15 tworzy już 7 ścieżek, a `index` jest jedną z nich

https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions